docs/index.html

```diff
@@ -1,66 +1,106 @@
+    autoHeader: true,
+    executeScript: true,
+    noEmoji: false,
+    mergeNavbar: true,
+
+    // Plugins
+    "flexible-alerts": {
+      style: "flat",
+    },
+    copyCode: {
+      buttonText: "Copy this snippet",
+      errorText: "Could not copy!",
+      successText: "Copied to clipboard",
+    },
+
+    // Theme
+    darklightTheme: {
+      siteFont: 'Lato',
+      defaultTheme: 'dark',
+      codeFontFamily: 'Space Mono',
+      bodyFontSize: '18px',
+      dark: {
+        accent: '#FE5A01',
+        toogleBackground: '#ffffff',
+        background: '#232328',
+        textColor: '#b4b4b4',
+        codeTextColor: '#ffffff',
+        codeBackgroundColor: '#313136',
+        borderColor: '#0d2538',
+        blockQuoteColor: '#FE5A01',
+        highlightColor: '#FE5A01',
+        sidebarSublink: '#b4b4b4',
+        codeTypeColor: '#ffffff',
+        coverBackground: 'linear-gradient(to left bottom, hsl(118, 100%, 85%) 0%,hsl(181, 100%, 85%) 100%)',
+        toogleImage: 'url(https://cdn.jsdelivr.net/npm/docsify-darklight-theme@latest/icons/sun.svg)'
       },
-      maxLevel: 2,
-    };
-  </script>
-  <!-- Docsify v4 -->
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/docsify/4.12.1/docsify.min.js"></script>
-  <script src="https://cdn.jsdelivr.net/npm/docsify-plugin-flexible-alerts@1"></script>
-  <script src="https://unpkg.com/docsify-copy-code@2"></script>
-  <!-- Prism (syntax highlighting) -->
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.25.0/components/prism-typescript.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.25.0/components/prism-markdown.min.js"></script>
-</html>
+      light: {
+        accent: '#FE5A01',
+        toogleBackground: '#091a28',
+        background: '#ffffff',
+        textColor: '#34495e',
+        codeTextColor: '#525252',
+        codeBackgroundColor: '#f8f8f8',
+        borderColor: 'rgba(0, 0, 0, 0.07)',
+        blockQuoteColor: '#FE5A01',
+        highlightColor: '#FE5A01',
+        sidebarSublink: '#222222',
+        codeTypeColor: '#091a28',
+        coverBackground: 'linear-gradient(to left bottom, hsl(118, 100%, 85%) 0%,hsl(181, 100%, 85%) 100%)',
+        toogleImage: 'url(https://cdn.jsdelivr.net/npm/docsify-darklight-theme@latest/icons/moon.svg)'
+      }
+    }
+  };
+</script>
+<!-- Docsify v4 -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/docsify/4.12.1/docsify.min.js"></script>
+<script src="//cdn.jsdelivr.net/npm/docsify-darklight-theme@latest/dist/index.min.js" type="text/javascript">
+</script>
+<script src="https://cdn.jsdelivr.net/npm/docsify-plugin-flexible-alerts@1"></script>
+<script src="https://unpkg.com/docsify-copy-code@2"></script>
+<!-- Prism (syntax highlighting) -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.25.0/components/prism-typescript.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.25.0/components/prism-markdown.min.js"></script>
+
+</html>
```

Comment:
I think this needs an endline? 😅
```suggestion
</html>

```